frontend/src/metabase/parameters/components/WidgetStatusIcon.tsx

```diff
@@ -22,6 +23,11 @@ export function WidgetStatusIcon({ name, onClick }: Props) {
   };
 
   return (
-    <Icon name={name} onClick={handleOnClick} size={12} className={classes} />
+    <Icon
+      name={name}
+      onClick={handleOnClick}
+      size={size ?? 12}
```

Comment:
This could be a default prop value

Reply:
143af9fc0429f3bd3923ab21e2be2aa21e0e0286